.github/workflows/ci.yml

```diff
@@ -296,8 +316,20 @@ jobs:
     steps:
       - name: "Check out the repo"
         uses: "actions/checkout@v3"
+
+      - name: "Install Pnpm"
+        uses: "pnpm/action-setup@v2"
+        with:
+          version: "8"
+
+      - name: "Install Node.js"
+        uses: "actions/setup-node@v3"
         with:
-          submodules: "recursive"
+          cache: "pnpm"
+          node-version: "lts/*"
+
+      - name: "Install the Node.js dependencies"
+        run: "pnpm install"
```

Comment:
This step is now redundant given that we are caching `node_modules`.

Reply:
the `slither-analyze` job is going to be removed from `ci.yml` and we will need the `pnpm install`in its own file, see #726

Reply:
Fine. In this case, it doesn't really matter what we do here - can you keep my latest commit as is, please?